src/wp-includes/media.php

```diff
@@ -939,7 +949,9 @@ function wp_get_registered_image_subsizes() {
  * @param int          $attachment_id Image attachment ID.
  * @param string|int[] $size          Optional. Image size. Accepts any registered image size name, or an array of
  *                                    width and height values in pixels (in that order). Default 'thumbnail'.
- * @param bool         $icon          Optional. Whether the image should fall back to a mime type icon. Default false.
+ * @param array|bool   $args          Optional. Additional arguments for the image to return. For backward
+ *                                    compatibility boolean can be used to determine whether the image should
+ *                                    fall back to a mime type icon. Default false.
```

Comment:
Please review other WordPress core functions that accept `$args` for how they document all individual supported keys for the array. We need to follow that here and thus expand this parameter documentation.